src/legacy/ui/public/table_info/table_info.html

```diff
@@ -1,6 +0,0 @@
-<div class="kuiPanel kuiPanel--centered kuiPanel--withToolBar">
-  <div
-    class="kuiTableInfo"
```

Comment:
I've seen this `kuiTableInfo` class name in a couple of more places, I wonder where does it come from.

Reply:
It's the old design we had before EUI